web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -17,6 +17,7 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.io.StringReader;
+import java.nio.file.Paths;
```

Comment:
Please remove changes to this file for now. Its also a pretty crucial class that seems to do a lot of intricate path manipulation (concatenations, replacements of separators, ...). I would like to have a closer look into this one as well.